schedule/schedule-appengine/src/main/client/jasify/admin/activity/admin-activites.js

```diff
@@ -28,8 +29,12 @@
         vm.viewSubscribers = viewSubscribers;
         vm.addSubscriber = addSubscriber;
 
+        vm.queryActivities = queryActivities;
+
         var $translate = $filter('translate');
 
+        vm.queryActivities();
```

Comment:
I understand why you moved this logic from the route/resolve to here.  The side effect is that the template renders and waits for the controller.  I'm curious as to what the difference is if the query is slow to respond.